File: Feitos em C Sharp/2807.cs
using System;
namespace ConsoleApp12 {
    class Program {
        static void Main(string[] args) {
			int[] vet = new int[46];
			int cont = 0;
			for (int a = 0; a < 46; a++) {
				vet[a] = 0;
			}
			vet[1] = 1;
			for (int a = 2; a < 46; a++) {
				vet[a] = vet[a - 1] + vet[a - 2];
			}
			int num = int.Parse(Console.ReadLine());
			for (int a = num; a > 0; a--) {
				if (a > 1) {
					Console.Write(vet[a] + " ");
				}
				else {
					Console.WriteLine(vet[a]);
				}
				cont++;
			}
		}
    }
}

File: Feitos em C/2807.c
#include <stdio.h>
int main() {
    int vet[46],num,cont=0,a;
    for(a=0;a<46;a++){
    	vet[a]=0;
	}
	vet[1]=1;
	for(a=2;a<46;a++){
    	vet[a]=vet[a-1]+vet[a-2];
	}
	scanf("%d",&num);
	for(a=num;a>0;a--){
		if(a>1){
			printf("%d ",vet[a]);
		}
		else{
			printf("%d\n",vet[a]);
		}
		cont++;
	}
    return 0;
}
